### DISCUSSION TITLE
Budibase Version Rollback Using Docker Pull?

### DISCUSSION THREAD

**Discussion by CovantaDataManagement** [Budibase/budibase#6353] [Help] [answered]
I accidentally hit the upgrade button on our development environment last night.  What is the docker command to pull version 1.0.46?  I have a separate sandbox to test the upgrades and as I feared, many things are broken after upgrade.  We need to go back to 1.0.46.

**Comment by Rory-Powell** [comment, marked as answer]
Hi @CovantaDataManagement, sorry to hear you're having issues with the latest version. I suspect this will primarily be caused by the changes to the docker compose file required when we migrated from envoy proxy to nginx. 

There are details on how to perform this change here:
https://github.com/Budibase/budibase/discussions/5169

For you current setup you can specifically target the server and worker at the version you mentioned above, like so:

<img width="1187" alt="Screenshot 2022-06-16 at 14 29 24" src="https://user-images.githubusercontent.com/8755148/174080758-e258d461-5188-4e77-a21c-2326b74fb9f1.png">

**Reply by CovantaDataManagement** [reply]
**### _THAT WORKED BEAUTIFULLY!!!!!! THANK YOU!!!!!_**
